### PR TITLE
bump: Samples to Akka Projection 1.4.0

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

**Review by pvlugter — APPROVED** [review]

File: samples/grpc/shopping-analytics-service-java/pom.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <akka.version>2.8.1</akka.version>
-        <akka-projection.version>1.4.0-M4</akka-projection.version>
+        <akka-projection.version>1.4.0</akka-projection.version>
         <akka-persistence-r2dbc.version>1.1.0</akka-persistence-r2dbc.version>
         <akka-management.version>1.2.0</akka-management.version>
```
Comment (patriknw) [inline samples/grpc/shopping-analytics-service-java/pom.xml:23]: bump Akka management at same time or separate PR?

File: samples/replicated/shopping-cart-service-scala/build.sbt [hunk 2]
```diff
@@ -32,7 +32,7 @@ val AkkaHttpVersion = "10.5.0"
 val AkkaManagementVersion = "1.3.0"
```
Comment (patriknw) [inline samples/replicated/shopping-cart-service-scala/build.sbt:32]: and Akka HTTP 10.5.2

Comment (patriknw) [reply samples/replicated/shopping-cart-service-scala/build.sbt:32]: and Akka 2.8.2

**Review by patriknw — COMMENTED** [review]
good, but might want to bump some more...

**Comment by johanandren** [comment]
Let's bump it all, I'll go over them another round so they align with bom/release train

**Pull request merged**